frontend/lib/routing.dart

```diff
@@ -31,4 +58,16 @@ final appRouter = GoRouter(
 class AppRoutes {
   static const snaps = '/snaps';
   static const debs = '/debs';
+
+  static FamilyName familyFromContext(BuildContext context) {
+    final route = GoRouterState.of(context).fullPath!;
+
+    if (route.startsWith(snaps)) {
+      return FamilyName.snap;
+    } else if (route.startsWith(debs)) {
+      return FamilyName.deb;
+    } else {
+      throw Exception('Unknown route: $route');
```

Comment:
if-else statements look fine for now since we have only 2 families. but isn't it better to use switch-case because in future we may have more families (image, for instance)?

Reply:
I couldn't use switch statement with `startsWith` method

Reply:
ah, definitely! no problem in this case